_episodes/09-documentation.md

```diff
@@ -412,4 +412,45 @@ conda:
 Commit to these changes and push to your repository. You should now see your documentation strings.
 
 
-{% include links.md %}
\ No newline at end of file
+#### Automatically Generating Documentation Using Sphinx-AutoAPI
+Sphinx includes an alternative to autodoc/autosummary that will automatically generate an "API Reference" page based
+on the docstrings throughout the projet's source code upon compiling using `make html`.
+This is [Spinx-AutoAPI](https://github.com/readthedocs/sphinx-autoapi). To install Sphinx-AutoAPI, use `pip`:
+
+``pip install sphinx-autoapi``
```

Comment:
Should use code block/syntax highlighting here. For this command, it would be

```
~~~
pip install sphinx-autoapi
~~~
{: .language-bash}
```

Reply:
Added in latest commit.